pkgs/development/python-modules/pyspark/default.nix

```diff
@@ -17,6 +17,14 @@ buildPythonPackage rec {
     substituteInPlace setup.py --replace py4j==0.10.7 'py4j>=0.10.7,<0.11'
   '';
 
+  postFixup = ''
+    # Original python script is wrapped, also make sure that python search path
+    # includes SPARK_HOME
+    substituteInPlace $out/bin/find-spark-home \
+        --replace 'export SPARK_HOME=$($PYSPARK_DRIVER_PYTHON "$FIND_SPARK_HOME_PYTHON_SCRIPT")' \
+                  'export SPARK_HOME=$("$FIND_SPARK_HOME_PYTHON_SCRIPT")''\n  export PYTHONPATH="$(dirname $SPARK_HOME):$PYTHONPATH"'
```

Comment:
Please avoid exporting `PYTHONPATH`.

Reply:
@FRidh Is it OK to modify an existing `PYTHONPATH` export in `$out/bin/pyspark` (latest commit) rather than adding a new one in `find-spark-home`? I can't think of a better way to change the search path.